src/pages/Subscribe/Subscribe.js

```diff
@@ -1,7 +1,362 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { useNavigate, useLocation } from 'react-router-dom';
 
 const Subscribe = () => {
-  return <h1>Subscribe</h1>;
+  const { pathname } = useLocation();
+
+  const [index, setIndex] = useState(1);
+  const [name, setName] = useState('');
+  const [address, setAddress] = useState('');
+  const PRICE = 39000;
+
+  const result = PRICE * index;
+
+  const navigete = useNavigate();
+
+  const goToBack = () => {
+    navigete(-1);
+  };
+
+  const onChangeName = e => {
+    setName(e.target.value);
+  };
+  const onChangeAddress = e => {
+    setAddress(e.target.value);
+  };
+
+  const onIndex = e => {
+    setIndex(e.target.value);
+  };
+
+  const onClickNaverBtn = e => {
+    alert('네이버페이로 결제하시겠습니까?');
+  };
+
+  const onClickKakaoBtn = e => {
+    alert('카카오페이로 결제하시겠습니까?');
+  };
+
+  const onClickCardBtn = e => {
+    alert('신용카드로 결제하시겠습니까?');
+  };
```

Comment:
```suggestion

const puchaseNotice = (e)=>{
alert(`${e.target.innerHTML}로 결제하시겠습니까?`)
}
```
이 함수로 3가지 버튼에 활용 해 보세요